### DISCUSSION TITLE
Write to inverter with solarman

### DISCUSSION THREAD

**Discussion by ballonchef** [StephanJoubert/home_assistant_solarman#647] [Q&A]
Dear all,

I successfully connected HO to mit ZCS Azzurro HYD 15000 using the sofar_g3hyd.yaml.

Does anyone know if and how I can write to my inverter? In my case, I would like to change the battery storage mode from "self-use" to "passive" and back.
Referring to the registers of the protocol, this should be possible:

![image_2024-08-17_183146840](https://github.com/user-attachments/assets/68e01ad0-43b2-4553-81dd-db01e41c7d33)

**Comment by bjeli1980** [comment]
My inverter is HYD 5K-EP and I want to do the same. Did you finally get it?
